schema/me/conversation/index.js

```diff
@@ -190,7 +190,28 @@ export const ConversationFields = {
   last_message_open: {
     type: GraphQLString,
     description: "Timestamp if the user opened the last message, null in all other cases",
-    resolve: conversation => null, // eslint-disable-line no-unused-vars
+    resolve: (conversation, options, request, { rootValue: { accessToken } }) => {
+      const radiationMessageId = get(conversation, "_embedded.last_message.radiation_message_id")
+      const impulseParams = {
+        conversation_id: conversation.id,
+        radiation_message_id: radiationMessageId,
+        "expand[]": "deliveries",
+      }
+      return gravity.with(accessToken, { method: "POST" })("me/token", {
+        client_application_id: IMPULSE_APPLICATION_ID,
+      }).then(data => {
+        return impulse.with(data.token, { method: "GET" })(
+          `message_details`,
+          impulseParams
+        ).then(({ message_details }) => {
+          if (message_details.length === 0) {
+            return null
+          }
+          const relevantDelivery = message_details[0].deliveries.find(d => d.email === conversation.from_email)
```

Comment:
It really shouldn't, end-users won't be accessing the inbox until we (finish the inbox) but also incorporate it into a home/nav update, plus some other updates in the app. I _think_ the last date I heard was Q1 next year (for a new version of the app to the store), which would be including the inbox. (our goal tho is to finish and 'ship' the inbox and move on to other things this quarter, for sure).